src/makeCalculator.js

```diff
@@ -4,7 +4,41 @@
  * @return {object}
  */
 function makeCalculator() {
-  // write code here
+  const calculator = {
+    result: 0,
+
+    add(item) {
+      this.result += item;
+    },
+
+    subtract(item) {
+      this.result -= item;
+    },
+
+    multiply(item) {
+      this.result *= item;
+    },
+
+    divide(item) {
+      if (item !== 0) {
+        this.result /= item;
+      }
```

Comment:
The check for division by zero is good, but consider throwing an error or logging a message to inform the user that division by zero is not allowed, as silently failing might lead to debugging difficulties.